### PR TITLE
Add new Conan tribe members

### PR DESCRIPTION
**Pull request by uilianries**
Adding new Conan tribe members.

We need to wait some profile pictures before merging

Related to https://github.com/conan-io/tribe/pull/22

/cc @memsharded

### CODE REVIEW

File: src/js/3_tribe-members.js [hunk 1]
```diff
@@ -330,5 +330,53 @@ var tribeMembers = [
...
     description: "Robotics software developer at NASA-JPL leveraging Conan to develop C/C++ manipulation applications",
     company: 'NASA-JPL'
-  }
+  },
+  {
+    name: 'Ihar Shashura',
+    image: 'Ihar Shashura.jpg',
+    description: "Ihar Povedano is currently working as a team/tech lead on cross-platform libraries.",
+    company: 'Self-Employed'
+  },
```
Comment (memsharded) [inline src/js/3_tribe-members.js:339]:
```suggestion
    image: 'Ihar_Shashura.jpg',
    description: "I am cross-platform (Android, iOS, Linux, macOS, Windows) engineer who passionate about cross-platform solutions based on c++. Had experience with different embedded devices, CAD systems, high-load servers, Video/Audio applications, financial software. M. Sc. in Math. ",
  },
```

**Review by memsharded — COMMENTED** [review]
I started to review, but it seems it is still half way, seems most data is outdated. Please ping me when it is ready for review.

File: src/js/3_tribe-members.js [hunk 2]
```diff
@@ -330,5 +330,60 @@ var tribeMembers = [
...
+    company: 'Train IT',
+    twitter: 'https://twitter.com/mateusz_pusz/',
+    linkedin: 'https://www.linkedin.com/in/mpusz/'
+  },
+  {
+    name: 'Udo Tremel',
+    description: "Principal R&D Engineer working on the geometry acquisition for meshing and the exposure of meshing technology in Ansys products.",
+    company: 'ANSYS',
+    twitter: 'https://twitter.com/mateusz_pusz/',
+    linkedin: 'https://www.linkedin.com/in/mpusz/'
```
Comment (memsharded) [inline src/js/3_tribe-members.js:372]:
```suggestion
```

This is wrong, please double check

**Review by memsharded — CHANGES_REQUESTED** [review]

File: src/js/3_tribe-members.js (hunk 2 above)
Comment (uilianries) [reply src/js/3_tribe-members.js]: Done

**Review by memsharded — APPROVED** [review]

**Pull request merged**